app/controllers/admin/visit_request/check_in_controller.rb

```diff
@@ -5,7 +5,11 @@ class CheckInController < ::Admin::BaseController
 
       def show
```

Comment:
On visit requests page we have real-time updating (actioncable), and in that case, you need to respond correctly. As it is in `ToggleVisitController` and other similar.

Reply:
Totally forgot about it. Thanks 🙇 
Will use `ToggleVisit` endpoint then